### PR TITLE
Added row for DKE feature available in Beta

### PR DESCRIPTION
**Pull request by pefrem**

### CODE REVIEW

File: microsoft-365/compliance/sensitivity-labels-versions.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ The numbers listed are the minimum Office application versions required for each
...
 |[Default sublabel for parent label](sensitivity-labels-office-apps.md#specify-a-default-sublabel-for-a-parent-label)|Current Channel: 2302+ <br /><br> Monthly Enterprise Channel: 2302+ <br /><br> Semi-Annual Enterprise Channel: 2302+ |Under review |Under review |Under review |Under review |
 |[Scope labels to files or emails](sensitivity-labels-office-apps.md#scope-labels-to-just-files-or-emails) |Current Channel: 2301+ <br /><br> Monthly Enterprise Channel: Under review <br /><br> Semi-Annual Enterprise Channel: Under review |16.69+ |Preview: Rolling out to [Beta Channel](https://insider.office.com/join/ios) |Preview: Rolling out to [Beta Channel](https://insider.office.com/join/android)| [Yes - opt-in](sensitivity-labels-sharepoint-onedrive-files.md) |
+|[Encrypt files using double key encryption](https://learn.microsoft.com/en-us/microsoft-365/compliance/double-key-encryption) |Preview: [Beta Channel](https://office.com/insider) <br /><br> Current Channel: Under review <br /><br>Monthly Enterprise Channel: Under review <br /><br> Semi-Annual Enterprise Channel: Under review |Under review |Under review |Under review| Under review |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-versions.md:69]:
```suggestion
|[Encrypt files using double key encryption](/microsoft-365/compliance/double-key-encryption) |Preview: [Beta Channel](https://office.com/insider) <br /><br> Current Channel: Under review <br /><br>Monthly Enterprise Channel: Under review <br /><br> Semi-Annual Enterprise Channel: Under review |Under review |Under review |Under review| Under review |
```

File: microsoft-365/compliance/sensitivity-labels-versions.md [hunk 2]
```diff
@@ -102,6 +104,8 @@ The numbers listed are the minimum Office application versions required for each
...
 |[Preventing oversharing as DLP policy tip](dlp-create-deploy-policy.md#scenario-2-show-policy-tip-as-oversharing-popup-preview)|Preview: [Current Channel (Preview)](https://office.com/insider) |Under review |Under review |Under review |Under review |
 |[Label inheritance from email attachments](sensitivity-labels-office-apps.md#configure-label-inheritance-from-email-attachments) |Current Channel: 2303+ <br /><br> Monthly Enterprise Channel: 2304+ <br /><br> Semi-Annual Enterprise Channel: 2302+ |Under review |Under review |Under review |Under review |
+|[Encrypt emails using double key encryption](https://learn.microsoft.com/en-us/microsoft-365/compliance/double-key-encryption) |Preview: [Beta Channel](https://office.com/insider) <br /><br> Current Channel: Under review <br /><br>Monthly Enterprise Channel: Under review <br /><br> Semi-Annual Enterprise Channel: Under review |Under review |Under review |Under review| Under review |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-versions.md:107]:
```suggestion
|[Encrypt emails using double key encryption](/microsoft-365/compliance/double-key-encryption) |Preview: [Beta Channel](https://office.com/insider) <br /><br> Current Channel: Under review <br /><br>Monthly Enterprise Channel: Under review <br /><br> Semi-Annual Enterprise Channel: Under review |Under review |Under review |Under review| Under review |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pefrem, kindly commit the suggested changes. Many thanks.

**Review by alejandrosMSFT — APPROVED** [review]

**Comment by cabailey** [comment]
This should have mentioned me - do not merge until checked

**Comment by cabailey** [comment]
Closing - picking up needed update for WXP with internal PR 20670

**Pull request closed without merging**